editor/plugins/org.fusesource.ide.camel.editor/src/org/fusesource/ide/camel/editor/properties/rest/RestVerbElementPropertiesSection.java

```diff
@@ -85,15 +82,21 @@ private AbstractParameterPropertyUICreator handleField(final Parameter p, final
 		AbstractParameterPropertyUICreator creator = createPropertyFieldEditor(page, p);
 		if (creator != null) {
 			IObservableValue<?> modelValue = (IObservableValue<?>) creator.getBinding().getModel();
+			modelValue.addChangeListener(event -> refreshClassAndBeanRefBindings());
 			IObservableValue<?> targetValue = (IObservableValue<?>) creator.getUiObservable();
-			modelValueMap.put(p.getName(), modelValue);
-			targetValueMap.put(p.getName(), targetValue);
+			targetValue.addChangeListener(event -> refreshClassAndBeanRefBindings());
+			// read only for now
+			creator.getControl().setEnabled(false);
```

Comment:
I think there was a problem with the rebase, this code was removed to have properties editable for rest operations